pkgs/desktops/gnome/extensions/extensionOverrides.nix

```diff
@@ -49,6 +49,7 @@ super: lib.trivial.pipe super [
     nativeBuildInputs = [ gobject-introspection wrapGAppsHook ];
     buildInputs = [ vte ];
     postFixup = ''
+      substituteInPlace "$out/share/gnome-shell/extensions/ddterm@amezin.github.com/bin/com.github.amezin.ddterm" --replace "gjs" "${gjs}/bin/gjs"
```

Comment:
This could also be `--prefix PATH` or `--suffix PATH` but [the `makeWrapper` docs](https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper) seem to suggest preferring `substituteInPlace`:

> \# Be advised that paths often should be patched in directly
> \# (via string replacements or in `configurePhase`).

Reply:
Oh I didn't know that, thanks